seqr/views/apis/users_api.py

```diff
@@ -1,5 +1,10 @@
+from __future__ import unicode_literals
+from future.standard_library import install_aliases
+install_aliases()
```

Comment:
what does this do? If it is something we need, can you add a comment here explaining it?

Reply:
It installs the aliases for `urllib.parse` for Python 2.
In Python 3, some of the methods of `urllib` have been moved under `urllib.parse`. For example, `urllib.quote_plus()` becomes `urllib.parse.quote_plus()`. The `future` package provides an alias of `urllib.parse` to support Python 2&3 compatible code.

Reply:
I will add comments to the source file.